main.js

```diff
@@ -194,18 +203,81 @@ function createWindow () {
   });
 }
 
+function createTrayIcon() {
+  logger.info('Creating tray icon');
+
+  // TODO: better to not use a hardcoded icon file
```

Comment:
What would you recommend here instead of a hardcoded path?

Reply:
Sorry, I should have detailed this better. Having multiple icon styles (light, dark, colour) for the tray and allowing the user to choose in the options would be great for theme integration. But probably this is not the most urgent issue.